aviary/interface/test/test_height_energy_mission.py

```diff
@@ -252,6 +253,48 @@ def test_custom_phase_builder_error(self):
                 optimizer='SLSQP',
             )
 
+    def test_support_constraint_aliases(self):
+        # Test specification of multiple constraints on a single variable.
+        modified_phase_info = self.phase_info.copy()
```

Comment:
do we want to use a deep copy here? I think that's what we have done most of the time we have made a copy of the phase_info

Reply:
Yeah, it wouldn't hurt to be careful.